components/coap/port/coap_notls.c

```diff
@@ -0,0 +1,190 @@
+/*
+* coap_notls.c -- Stub Datagram Transport Layer Support for libcoap
```

Comment:
(One doubt) This file will eventually move to `libcoap` going ahead, right?

Reply:
Yes, once https://github.com/obgm/libcoap/pull/353 is merged and then libcoap is updated in esp-idf, there will be no need for port/coap_notls.c or port/coap_debug.c.

Once these MbedTLS changes to port/coap_mbedtls.c are merged back into libcoap (which I am working on), there will be no need for port/coap_mbedtls.c when libcoap is updated in esp-idf.

Reply:
Sounds good, thanks.